docs/running-notebooks/opening-sessions/index.md

```diff
@@ -14,6 +14,12 @@ It could take several minutes to start the interactive session.
 Please note that it will be automatically closed after 7 days of inactivity.
 ```
 
+By clicking on the link you will have access to a Jupyter notebook interface:
+
+![jupyter-notebook](../../images/interactive-session-jupyter-notebook.png){.screenshot-browser-mockup}
+
+Keep in mind that starting the notebook might take a minute, if you see "Service unavailable", just wait and retry.
```

Comment:
After seeing the updated version, I would delete the "Keep in mind" phrase, because it disturbs the reading flow.  We are already warning about this in the console output shown above, so it should not be necessary  to add it also here!